src/chigame/games/views.py

```diff
@@ -414,6 +421,30 @@ def check_guess(request, pk):
     )
 
 
+class TournamentBracketView(TemplateView):
```

Comment:
Would like to see the bracket view as a button somewhere on the detail view, so users can access it